### DISCUSSION TITLE
iPXE for Raspi ARMv6 32-bit

### DISCUSSION THREAD

**Discussion by cmonty14** [ipxe/ipxe#1212] [General]
Hi,
In my setup with a (ISP) router that provides NAS functionality over SAMBA it would make sense to deploy iPXE on a Raspi.
Hereby I could offer PXE network boot and diskless boot for any client in my network.

Could you consider to build iPXE for ARMv6 (32-bit)?

THX

**Comment by NiKiZe** [comment]
What do you want to boot exactly? Note that iPXE is the bootloader that runs on the device you want to boot. In a full network boot environment it is loaded over TFTP when the client is instructed to do so by information in DHCP. 

The 32 bit Pis are not something that makes sense to network boot.

**Reply by cmonty14** [reply]
I'm sorry, my understanding of iPXE was wrong.
My use-case is to serve PXE on Raspi that allows other clients to boot over network.
A simple solution that provides this functionality is [iVentoy](https://iventoy.com/), that is using iPXE somehow.
However iVentoy is only available on x86_64.

**Reply by NiKiZe** [reply]
https://ipxe.org/appnote/proxydhcp
This is the DHCP part, and TFTP 
But before this, try to boot iPXE from a usb flash device, take one step at a time.
